services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/organisaatio/Persistence.kt

```diff
@@ -0,0 +1,19 @@
+package fi.hel.haitaton.hanke.organisaatio
+
+import org.springframework.data.jpa.repository.JpaRepository
+import java.time.LocalDateTime
+import javax.persistence.*
+
+@Entity @Table(name = "organisaatio")
```

Comment:
Same comment here as with @tintotti: why fill constructors with all fields in class? If all have default values like here then it might be okay, but still.. if this was coded in Java, would they still be all in constructor?